src/app/tests/TestReadInteraction.cpp

```diff
@@ -1575,6 +1572,7 @@ void TestReadInteraction::TestSubscribeRoundtrip(nlTestSuite * apSuite, void * a
         ctx.DrainAndServiceIO();
 
         NL_TEST_ASSERT(apSuite, delegate.mGotReport);
+        NL_TEST_ASSERT(apSuite, delegate.mGotEventResponse == true);
```

Comment:
Shouldn't we be clearing this earlier before checking it here?

Reply:
Yes, Done, thanks